setup.py

```diff
@@ -6,8 +6,8 @@
 setuptools.setup(
     name="agentFIL",
     version="0.1",
-    author="Kiran Karra, Tom Mellan",
-    author_email="kiran.karra@gmail.com, t.mellan@imperial.ac.uk",
+    author="Kiran Karra, Tom Mellan, Juan",
```

Comment:
sorry I forgot to include you. Can you put your last name also just so we can be consistent, then we can merge this asap.